native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/sender/states/http2/SendingRstFrame.java

```diff
@@ -0,0 +1,108 @@
+package io.ballerina.stdlib.http.transport.contractimpl.sender.states.http2;
```

Comment:
Missing licence header

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-http/pull/659/commits/9f6bd071ebb238a6ad2f5417bcca03cbd9304737